Documentation/Books/Manual/ReleaseNotes/KnownIssues33.md

```diff
@@ -0,0 +1,13 @@
+Known Issues
+============
+
+This page lists some known issues of the ArangoDB suite of products. 
+
+ArangoDB Customers can access the [_Technical & Security Alerts_](https://arangodb.atlassian.net/wiki/spaces/DEVSUP/pages/223903745) page after logging in the Support Portal.
+
+| # | Issue      |
+|---|------------|
+| 4 | **Date Added:** 2018-11-30 <br> **Component:** arangod <br> **Deployment Mode:** All <br> **Description:** Wrong suggestion printed in the log on how to optimize an OS setting, if followed, could cause ArangoDB to run into problems as the number of memory mappings will keep growing <br> **Affect Versions:** 3.3.0 to 3.3.19 <br> **Fixed in Versions:** 3.3.20 <br> **Reference:** [Link for ArangoDB Customers](https://arangodb.atlassian.net/plugins/servlet/servicedesk/customer/confluence/shim/spaces/DEVSUP/pages/228622337/Technical+Alert+%232%3A+Set+Linux+variable+overcommit_memory+to+0+or+1)  |
```

Comment:
Redirects to KB page, so we can use shorter link can't we?

Should it be *ArangoDB Server* instead of  `arangod` as component?

```suggestion
| 4 | **Date Added:** 2018-11-30 <br> **Component:** arangod <br> **Deployment Mode:** All <br> **Description:** Wrong suggestion printed in the log on how to optimize an OS setting, if followed, could cause ArangoDB to run into problems as the number of memory mappings will keep growing <br> **Affect Versions:** 3.3.0 to 3.3.19 <br> **Fixed in Versions:** 3.3.20 <br> **Reference:** [Link for ArangoDB Customers](https://arangodb.atlassian.net/servicedesk/customer/kb/view/228622337)  |
```

Reply:
I would prefer we use arangod for the moment, as arangod can be also agency / coordinator etc